setup.py

```diff
@@ -25,7 +25,7 @@
         'tracery',
     ],
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
Looking good, if requiring https://pypi.org/project/pyjson5/1.6.7/, we should also set our min Python version to match that, which is 3.5+:

```suggestion
    python_requires='>3.5',
```
(Personally I think we should only support the same as upstream, [currently 3.9+](https://devguide.python.org/versions/), but that's another issue.)